apiBase/Model.es6.js

```diff
@@ -179,12 +180,16 @@ export default class Model {
     return fakeUUID();
   }
 
+  makePaginationId(data) {
```

Comment:
I want to make sure it has access to the raw api response, and that's not being stored. So making it a function was the easiest way to do it